### PR TITLE
add OutputStatistics facet

### PR DESCRIPTION
**Pull request by julienledem**
Signed-off-by: Julien Le Dem <julien@apache.org>

### CODE REVIEW

**Comment by mobuchowski** [comment]
I think if we're adding something to official json schema, we need to mention it in docs too.

File: spec/OpenLineage.json [hunk 1]
```diff
@@ -380,7 +383,29 @@
       ],
       "type": "object"
     },
-
+    "OutputStatisticsOutputDatasetFacet": {
```
Comment (mobuchowski) [inline spec/OpenLineage.json:386]: I think if we're adding facets to json schema, we need to mention them in the docs here too:

https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.md#standard-facets

Comment (julienledem) [reply spec/OpenLineage.json:386]: Yes! Thank you for pointing this out

Comment (julienledem) [reply spec/OpenLineage.json:386]: I just added it.

File: spec/OpenLineage.json [hunk 2]
```diff
@@ -380,7 +383,29 @@
...
+        },
+        {
+          "type": "object",
+          "properties": {
+            "rowCount": {
+              "description": "The number of rows written to the dataset",
+              "type": "integer"
+            },
+            "size": {
+              "description": "The size in mb written to the dataset",
```
Comment (collado-mike) [inline spec/OpenLineage.json:399]: Both the Spark and BigQuery integrations publish this in bytes

Comment (julienledem) [reply spec/OpenLineage.json:399]: thanks!

**Review by collado-mike — APPROVED** [review]

**Pull request merged**